bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelCommand.java

```diff
@@ -227,12 +227,27 @@
     TRACK_FORWARD("Track Forward", RotelConnector.PRIMARY_CMD, (byte) 0x09, "track_fwd", "trkf"),
     TRACK_BACKWORD("Track Backward", RotelConnector.PRIMARY_CMD, (byte) 0x08, "track_back", "trkb"),
     TRACK("Request current CD track number", null, "track"),
-    FREQUENCY("Request current frequency for digital source input", "get_current_freq", "freq"),
+    FREQUENCY("Request current frequency for digital source input", "get_current_freq", "freq?"),
     DISPLAY_REFRESH("Display Refresh", RotelConnector.PRIMARY_CMD, (byte) 0xFF),
-    DIMMER_LEVEL_GET("Request current front display dimmer level", "get_current_dimmer", "dimmer"),
+    DIMMER_LEVEL_GET("Request current front display dimmer level", "get_current_dimmer", "dimmer?"),
     DIMMER_LEVEL_SET("Set front display dimmer to level", "dimmer_", "dimmer_"),
     UPDATE_AUTO("Set Update to Auto", "display_update_auto", "rs232_update_on"),
-    UPDATE_MANUAL("Set Update to Manual", "display_update_manual", "rs232_update_off");
+    UPDATE_MANUAL("Set Update to Manual", "display_update_manual", "rs232_update_off"),
+    TCBYPASS_TOGGLE("Mute Toggle", RotelConnector.PRIMARY_CMD, (byte) 0x00, "bypass", "bypass"),
+    TCBYPASS_ON("Bypass On", "bypass_on", "bypass_on"),
+    TCBYPASS_OFF("Bypass Off", "bypass_off", "bypass_off"),
+    TCBYPASS("Request current tone bypass state", "get_bypass_status", "bypass?"),
```

Comment:
The ASCII V1 command does not exist. in V1, the command is for tone control status, so the reverse.
So I suggest to set the V1 command to a null string.